service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/cmo/AgreedCMOUploadedEventHandler.java

```diff
@@ -48,7 +48,7 @@ public void sendNotificationForAdmin(final AgreedCMOUploaded event) {
     public void sendNotificationForJudge(final AgreedCMOUploaded event) {
         CaseData caseData = event.getCaseData();
 
-        if (caseData.hasAllocatedJudgeEmail()) {
+        if (event.getHearing().getHearingJudgeLabel() != null || caseData.hasAllocatedJudgeEmail()) {
```

Comment:
should this condition not be 

`event.getHearing().getHearingJudgeLabel() != null && event.getHearing().getJudgeAndLegalAdvisor().getJudgeEmailAddress() != null `

Reply:
Have updated to a better condition